pom.xml

```diff
@@ -118,6 +118,14 @@
 
         <persistence.enabled>true</persistence.enabled>
         <clustering.enabled>true</clustering.enabled>
+        <!--
+            Number of instance replicas to start during initial setup. This value is currently only used by Bitbucket,
+            other products' values.yaml ignore this property. Other products need to perform additional setup steps on
+            a single node after Helm chart installation.
+        -->
+        <replica.count>2</replica.count>
```

Comment:
I made this change so the setup for Bitbucket becomes easier. It does not scale up the instance after deployment anymore and instead starts up with 2 nodes. All other products still work the same way, this variable is only used for the Bitbucket templates, as is mentioned in the comment above.